app/javascript/src/casa_case.js

```diff
@@ -84,7 +87,67 @@ function courtMandateHtml (index) {
   }
 }
 
+function showBtn (el) { el.classList.remove('d-none') }
```

Comment:
All of this javascript is directly moved from app/views/case_court_reports/index.html.erb for the purposes of reusing it. The only change is how I set the url on line 114